### PR TITLE
Added the MongoDB Database Driver Code

### PR DESCRIPTION
**Pull request by MattXLei**

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -9,6 +9,16 @@ const fs = require('fs');
 // load additional environment variables (ex: database and storage bucket passwords) from .env file
 env.config();
 
+// MongoDB database connection code
+const { MongoClient, ServerApiVersion } = require('mongodb');
+const uri = "mongodb+srv://admin:webarcade7@cluster0.rporgjs.mongodb.net/?retryWrites=true&w=majority";
```
Comment (rdeepak2002) [inline src/server.js:14]: It is not secure to store passwords in code. Store this in an environment variable called "MONGO_DB_URI", then retrieve it using process.env.MONGO_DB_URI

Add the following to your .env file: https://docs.google.com/document/d/1F6kL7zoONb92WJbcmlhfFXMjz2BXWTEyaqaB1gwTrb8/edit

**Review by rdeepak2002 — CHANGES_REQUESTED** [review]
Review comment

**Comment by MattXLei** [comment]
I fixed the document; you can review it now

**Review by rdeepak2002 — APPROVED** [review]
LGTM! Feel free to merge. Thank you for your work.

**Pull request merged**